data/leadership.js

```diff
@@ -85,8 +80,8 @@ export default [
 		img: '/images/officers/MuthuPalaniappan.jpg',
 	},
 	{
-		name: 'Amy Seo',
+		name: 'Ivy Kang',
 		position: 'Co-President, Design',
-		img: '/images/officers/AmySeo.png',
+		img: '/images/officers/IvyKang.JPG',
```

Comment:
All of the images we put in are compressed to reduce file size, could you please run her picture  through a compressor like https://imagecompressor.com/ ?

Reply:
Adding on top of this, the way that we format images of officers, we ideally would like the images to be of square shape, can you please edit Ivy's picture so that it's of square size? <img width="377" alt="Screenshot 2023-01-03 at 11 03 04 PM" src="https://user-images.githubusercontent.com/65370631/210501313-ff3bb65e-a8f6-420e-b5dd-44b5c5111bcd.png">

Reply:
Thank you for catching this!